lib/src/view/game/game_body.dart

```diff
@@ -329,6 +340,18 @@ class GameBody extends ConsumerWidget {
     );
   }
 
+  void _chatListener(
+    AsyncValue<ChatState>? prev,
+    AsyncValue<ChatState> state, {
+    required BuildContext context,
+    required WidgetRef ref,
+  }) {
+    if (prev == null || !prev.hasValue || !state.hasValue) return;
+    if (state.requireValue.unreadMessages > prev.requireValue.unreadMessages) {
+      ref.read(soundServiceProvider).play(Sound.genericNotify);
```

Comment:
I don't think we should use that sound. Because it is the same as the game start and game ending sounds it is confusing.

A better one is `confirmation`.

Also in order to make it less disturbing during a game we should lower the volume of this specific sound effect.

You can add another `volume` parameter to [the play method](https://github.com/lichess-org/mobile/blob/main/lib/src/model/common/service/sound_service.dart#L84) to that effect. Should be a number between 0 and 1 to apply to the master volume.